rskj-core/src/main/java/org/ethereum/vm/OpCode.java

```diff
@@ -262,6 +262,10 @@
      * (0x45) Get the block’s gas limit
      */
     GASLIMIT(0x45, 0, 1, BASE_TIER),
+    /**
+     * (0x47) Get the senders balance
+     */
+    SELFBALANCE(0x47, 0, 1, LOW_TIER),
```

Comment:
verify that selfbalance gas cost matches cost from [this](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1884.md) EIP 

> SELFBALANCE is priced as GasFastStep, at 5 gas.